.github/workflows/eslint.yml

```
@@ -52,9 +52,34 @@ jobs:
+                    "rules": []
+                  }
+                },
+                "results": []
+              }
+            ]
+          }
+          EOF
+          fi
+
       - name: Upload analysis results to GitHub
         uses: github/codeql-action/upload-sarif@b20883b0cd1f46c72ae0ba6d1090936928f9fa30 # v4.32.0
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)